tests/cli/main_tool.py

```diff
@@ -16,7 +16,11 @@ class MainToolTest(unittest.TestCase):
   def setUp(self):
     self.tool = dftimewolf_recipes.DFTimewolfTool()
     self.tool.LoadConfiguration()
-    self.tool.ReadRecipes()
+    try:
+      self.tool.ReadRecipes()
+    except KeyError:
```

Comment:
Are you still running into this problem despite deregistering the recipes in TearDown?

Reply:
Yes :(